src/router/index.ts

```diff
@@ -6,16 +6,19 @@ import SignUpPage from "@/pages/SignUpPage.vue";
 const routes: Array<RouteRecordRaw> = [
   {
     path: "/",
+    name: "main",
```

Comment:
This is good practice to add names for routes, we also will need to add names for other routes as well...